### ISSUE TITLE
Colliding paths

### ISSUE THREAD

**Issue by CMCDragonkai** [MatrixAI/Polykey#44]
There are colliding paths in the `dist` directory because on Windows, its filesystems are case insensitive.

```
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'dist/Polykey.js'
  'dist/polykey.js'
  'dist/Polykey.js.map'
  'dist/polykey.js.map'
```

This should not be done. It should either be `Polykey.js` or `polykey.js`. I think the lowercase makes more sense here if you're producing a library. But also within `dist` should be `dist/bin` for the CLI executables.

**Comment by CMCDragonkai** [comment]
Which PR fixes this? @robert-cronin

**Comment by robert-cronin** [comment]
Sorry for the late reply. This was fixed in PR #40 I believe. The duplicate paths arose because the cli and the lib had the same name and were compiled to the same directory. The cli is now named `pk.js` and is build to `dist/cli/pk.js`
